homeassistant/components/upnp/__init__.py

```diff
@@ -34,7 +34,7 @@
 NOTIFICATION_ID = "upnp_notification"
 NOTIFICATION_TITLE = "UPnP/IGD Setup"
 
-PLATFORMS = ["sensor"]
+PLATFORMS = ["binary_sensor", "sensor"]
```

Comment:
When you add a new platform to integrations you should also add yourself as a codeowner :+1:

Reply:
This should still be done, now you are not only adding a platform, but you are refactoring the integration as well :clap: